cmd/airplane/views/initcmd/init.go

```
@@ -53,17 +55,21 @@ func New(c *cli.Config) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&cfg.envSlug, "env", "", "The slug of the environment to query. Defaults to your team's default environment.")
+	cmd.Flags().StringVar(&cfg.from, "from", "", "Path to an existing github folder to initialize.")
 
 	return cmd
 }
 
 func run(ctx context.Context, cfg config) error {
-	if err := promptForNewView(&cfg); err != nil {
-		return err
-	}
-
-	if err := createViewScaffolding(&cfg); err != nil {
-		return err
+	if strings.HasPrefix(cfg.from, "github.com/") || strings.HasPrefix(cfg.from, "https://github.com/") {
+		return initWithExample(&cfg, cfg.from)
+	} else {
```

I would think that if you provide a from, we always init with example. If your from is malformed, we can throw an error in initWithExample. I think it's more confusing to just ignore if I have a typo